test/stubs.py

```diff
@@ -721,7 +721,7 @@ def delete_cache(self):
         self.server_status = None
 
 
-class StubAvailableEntitlementsCache(AvailableEntitlementsCache):
+class StubContentAccessModeCache(ContentAccessModeCache):
```

Comment:
Any reason to move all these stubs around? It could be an artefact of the cherry-pick but it doesn't seem meaningful to me.

Reply:
It was an artifact of the cherry-pick, yes.  Doesn't seem meaningful to me either.

Reply:
I reordered them